src/win32cfg.h

```diff
@@ -92,11 +98,7 @@
 #define SIZEOF_UNSIGNED_INT 4
 
 /* The size of `unsigned long', as computed by sizeof. */
-#ifdef _WIN64
-#define SIZEOF_UNSIGNED_LONG 8
-#else
 #define SIZEOF_UNSIGNED_LONG 4
-#endif
```

Comment:
If you mean the endif, then because I removed the corresponding ifdef (and else). If you mean the change in general, then because for both 32- and 64-bit MSVC, [long is 4 bytes](https://msdn.microsoft.com/en-us/library/s3f49ktz.aspx).